rcl_action/test/rcl_action/test_names.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright 2018 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include "rcl_action/names.h"
+
+#include "rcl/allocator.h"
+#include "rcl/types.h"
+
+
+TEST(TestActionNames, test_action_goal_service_name)
```

Comment:
It might be good to add tests for invalid arguments too.

Reply:
Agreed. I've refactored tests to avoid so much duplication.